Purchases/Purchasing/RCReceiptFetcher.swift

```diff
@@ -0,0 +1,41 @@
+//
+//  RCReceiptFetcher.swift
+//  Purchases
+//
+//  Created by Javier de Martín Gil on 8/7/21.
+//  Copyright © 2021 Purchases. All rights reserved.
+//
+
+import Foundation
+
+public class RCReceiptFetcher: NSObject {
+
+    @objc public var receiptData: NSData {
+
+        guard let receiptURL: URL = Bundle.main.appStoreReceiptURL else {
+            Logger.debug(Strings.receipt.unable_to_load_receipt)
+            return NSData()
+        }
```

Comment:
Don't know how you all want to deal with this. Don't know the specifics of when a receipt will be missing from a device. In this case it will return an empty `NSData` object and log the corresponding error. 

Any other suggestion here or any other approach?

Reply:
Okay, got it. Used `MockReceiptFetcher` as a blueprint on the method definition, didn't know it was Swift already.